<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>construtcor_page</class>
 <widget class="QWidget" name="construtcor_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0">
     <item>
      <widget class="QPushButton" name="prev_page_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Prev
page</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="questions_manage_layout" stretch="0,1,0,0">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="num_question_label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delete_page_button">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Delete page</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="questions_preview">
         <property name="frameShape">
          <enum>QFrame::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>4</number>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QVBoxLayout" name="questions_layout"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="add_question_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Add question</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="add_page_button">
         <property name="text">
          <string>Add page</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="next_page_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Next
page</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="done_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
